api/v1/app.py

```diff
@@ -16,6 +17,12 @@ def close(exit):
     storage.close()
 
 
+@app.errorhandler(404)
+def not_found(error):
+    response = jsonify({'error': 'Not found'})
+    return make_response(response, 404)
```

Comment:
Here you should return this response value directly.
```suggestion
    return make_response(jsonify({'error': 'Not found'}), 404)
```